src/sctools/platform.py

```diff
@@ -219,6 +224,105 @@ def merge_cell_metrics(cls, args: Iterable[str]=None) -> int:
         merge.execute()
         return 0
 
+    @classmethod
+    def bam_to_count_matrix(cls, args: Iterable[str]=None) -> int:
+        """Command line entrypoint for constructing a count matrix from a tagged bam file.
+
+        Constructs a count matrix from an aligned bam file sorted by cell barcode, molecule
+        barcode, and gene id.
+
+        Parameters
+        ----------
+        args : Iterable[str], optional
+            Arguments list, for testing (see test/test_entrypoints.py for example). The default
+            value of None, when passed to `parser.parse_args` causes the parser to
+            read `sys.argv`
+
+        Returns
+        -------
+        return_call : 0
+            return call if the program completes successfully
+
+        """
+        parser = argparse.ArgumentParser()
```

Comment:
[Irrelevant] Recently I found a lib called [Click](http://click.pocoo.org/5/), which is developed by the author of flask and werkzeug. I found it really elegant and maybe in the future it can be used to replace the parsers here.

Reply:
Click is nice for simple parsers but it doesn't work for quite a few more complex things (including inheritance). I prefer `argparse`, but agree that I'm not yet using the `argparse` options that I'm alluding to because we haven't expanded the library to have multiple different 3' pipelines. 

When we have nuclei, in-drop, and drop-seq, I suspect we'll want `argparse`. If that's wrong I will revisit. :)